translations/common_en.ts

```diff
@@ -481,11 +481,11 @@ Additional licenses and components</translation>
     </message>
     <message>
         <source>Key certificate sign</source>
-        <translation>Key certificate signature</translation>
+        <translation>Key certificate signing</translation>
     </message>
     <message>
         <source>CRL sign</source>
-        <translation>CRL signature</translation>
+        <translation>CRL signing</translation>
```

Comment:
I think this should remain "signature" (based on the estonian translation of "CRL allkiri")